package.json

```diff
@@ -196,6 +196,8 @@
   "license": "Apache-2.0",
   "scripts": {
     "lint": "eslint --ext .js,.ts,.tsx --max-warnings 0 . && tsc --noEmit --skipLibCheck",
+    "tsc:ci": "tsc --noEmit --skipLibCheck --project tsconfig.web.json && tsc --noEmit --skipLibCheck --project tsconfig.native.json",
```

Comment:
Can you pl break this into 2 commands, one web one native and then have `tsc:ci` run both?